docs/installation/config/openzaak_config.rst

```diff
@@ -301,3 +301,70 @@ endpoint is accessible at ``{{base_url}}/catalogi/api/v1/zaaktypen`` (where
     A GET request to the Catalogi API using Postman
 
 .. _Postman: https://www.getpostman.com/
+
+
+Using external ZGW APIs
+=======================
+
+Open Zaak provides production-grade :ref:`API's<api_index>` to store and retrieve data:
+
+* Zaken API (case instances)
+* Documenten API (documents)
+* Catalogi API (case types)
+* Besluiten API (decisions)
```

Comment:
Autorisaties API is not listed, nor do I think it's relevant to mention what OZ itself offers.

I'd rather see something along the lines of:

```rst
While Open Zaak itself implements all the necessary APIs for Zaakgericht Werken, it is possible
(conform the VNG standard) to use these APIs from _other components_ than Open Zaak that
also implement the VNG standard(s).

For example, you might use an alternative implementation of the Documenten API. Using this
example, this guide describes the necessary configuration steps in Open Zaak.
```

Reply:
The introduction is changed. Thank you for the provided text